webpack/JobWizard/steps/Schedule/StartEndDates.js

```diff
@@ -11,18 +11,24 @@ export const StartEndDates = ({
   setEnds,
   isNeverEnds,
   setIsNeverEnds,
+  setValid,
 }) => {
+  const [validEnd, setValidEnd] = useState(true);
   const toggleIsNeverEnds = (checked, event) => {
     const value = event?.target?.checked;
     setIsNeverEnds(value);
   };
-  const validateEndDate = () => {
-    if (isNeverEnds) return 'success';
-    if (!starts || !ends) return 'success';
-    if (new Date(starts).getTime() <= new Date(ends).getTime())
-      return 'success';
-    return 'error';
-  };
+  useEffect(() => {
+    if (isNeverEnds) setValidEnd(true);
+    else if (!starts || !ends) setValidEnd(true);
+    else if (new Date(starts).getTime() <= new Date(ends).getTime())
+      setValidEnd(true);
+    else setValidEnd(false);
+  }, [starts, ends, isNeverEnds]);
+  useEffect(() => {
+    setValid(validEnd);
```

Comment:
if eventually we update `setValid` with `validEnd`, do we really need to manage a state for `validEnd`? or can we just use `setValid` ?